app/views/catalog/_index_gallery.html.erb

```diff
@@ -0,0 +1,9 @@
+<% #OVERRIDE blacklight-gallery v0.12.0: change render thumbnail tag to use full_url for shared search URLs %>
```

Comment:
Comment syntax `<%#`

Reply:
I copied this directly from Blacklight gallery and forgot to change the format. Updating all of them now. :)